geometry_msgs/Pose pose
float64 v_arrive
float64 v_depart
